src/lib/components/WebvizPluginPlaceholder/webviz_plugin_component.css

```diff
@@ -116,3 +117,17 @@
     opacity: 1.0;
     color: black;
 }
+
+.webviz-config-plugin-button-important
+{
+    opacity: 1.0;
+    color: #ff9900 !important;
```

Comment:
Is the `!important` necessary here, or can we use other parts of the CSS specificity? https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity